### ISSUE TITLE
Question for pre-trained weights link.

### ISSUE THREAD

**Issue by kyhyk** [akshaydudhane16/Burstormer#10]
Hello, thank you for open source code!

I have trouble using pre-trained weights of Burst Super-resolution page.
There's no weights about back_projection.

I check the similar question posted earlier. 
Can you check one more time if it is right for Burstormer's pretrained weights?

Thanks.

**Comment by Xinyil256** [comment]
Hi! Have you solved this problem? I meet the same one.

> Hello, thank you for open source code!
> 
> I have trouble using pre-trained weights of Burst Super-resolution page. There's no weights about back_projection.
> 
> I check the similar question posted earlier. Can you check one more time if it is right for Burstormer's pretrained weights?
> 
> Thanks.

**Comment by kyhyk** [comment]
I just trained by myself...!

> Hi! Have you solved this problem? I meet the same one.
> 
> > Hello, thank you for open source code!
> > I have trouble using pre-trained weights of Burst Super-resolution page. There's no weights about back_projection.
> > I check the similar question posted earlier. Can you check one more time if it is right for Burstormer's pretrained weights?
> > Thanks.

**Comment by Magauiya** [comment]
I also encountered the same problem. @kyhyk did you achieve the same performance on validation set?

**Comment by cvkaiming** [comment]
> Hello, thank you for open source code!
> 
> I have trouble using pre-trained weights of Burst Super-resolution page. There's no weights about back_projection.
> 
> I check the similar question posted earlier. Can you check one more time if it is right for Burstormer's pretrained weights?
> 
> Thanks.

hello，may I ask if you reached the performance of the super-resolution Track2 in the article?
